README.md

```diff
@@ -0,0 +1 @@
+# svelte-template
```

Comment:
I do like when the README explains how to use the template. e.g. https://github.com/sveltejs/template

We can optionally use a degit configuration file to tell degit clients to ignore the README.

Reply:
Sure, I have no feelings or peeves around this one way or another at this juncture. Feel free to add what you feel would make this simple for others to get started.

Reply:
https://github.com/graph-paper-org/app-template/pull/9